### PR TITLE
Enable streaming OpenAI requests to work outside of Node

### PR DESCRIPTION
**Pull request by nfcampos**
Streaming requests now use our axios fetch adapter

### CODE REVIEW

File: langchain/src/util/axios-fetch-adapter.d.ts [hunk 1]
```diff
@@ -2,3 +2,14 @@
...
+
+export interface StreamingAxiosConfiguration {
+  responseType: "stream";
+
+  /**
+   * Called when a message is received. NOTE: Unlike the default browser
+   * EventSource.onmessage, this callback is called for _all_ events,
+   * even ones with a custom `event` field.
+   */
+  onmessage?: (ev: EventSourceMessage) => void;
```
Comment (agola11) [inline langchain/src/util/axios-fetch-adapter.d.ts:14]: nit: `onMessage` ?

File: langchain/src/chat_models/openai.ts [hunk 2]
```diff
@@ -258,68 +255,94 @@ export class ChatOpenAI extends BaseChatModel implements OpenAIInput {
...
+              ...params,
+              messages: messages.map((message) => ({
+                role: messageTypeToOpenAIRole(message._getType()),
+                content: message.text,
+              })),
+            },
+            {
+              responseType: "stream",
+              onmessage: (event) => {
+                if (event.data === "[DONE]") {
```
Comment (agola11) [inline langchain/src/chat_models/openai.ts:274]: could there ever be extra newlines or carriage returns here?

**Review by agola11 — APPROVED** [review]
lgtm, thanks!

File: langchain/src/chat_models/openai.ts (hunk 2 above)
Comment (nfcampos) [reply langchain/src/chat_models/openai.ts:274]: yea I guess it doesn't hurt to call .trim, will add

**Pull request merged**